src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -3804,11 +3805,8 @@ public class EventListener : IDisposable
         static EventListener()
         {
 #if FEATURE_PERFTRACING
-            // Ensure that NativeRuntimeEventSource is initialized so that EventListeners get an opportunity to subscribe to its events.
```

Comment:
Lets restore and update this comment. I assume the rationale is that we want EventListeners to be able to subscribe to events emitted by native code which may come before the first event emitted via managed code.